client/js/Ui/tutorial.js

```diff
@@ -0,0 +1,327 @@
+SCWeb.ui.Tutorial = {
+    inProgress: false,
+    activeElem: undefined,
+    tutorialNodeCreated: false,
+    tutorialEdgeCreated: false,
+
+    init: function() {
+        return new Promise(resolve => {
+            const self = this;
+            $(document)
+                .ready(() => {
+                    $('#abort-tutorial-button').click(() => {
+                        self.end()
+                    });
+                });
+            resolve();
+        });
+    },
+
+    initStages: function() {
+        this.tutorialStages = [
+            {
+                elemID: "#search-panel",
+                desc: {
+                    rus: {
+                        title: 'Поле поиска',
+                        text: 'Добро пожаловать в обучение! Здесь Вы узнаете об основном функционале системы. Обучение всегда можно завершить досрочно нажатием кнопки "Abort tutorial" в правом нижнем углу этого окна. Ваше первое задание - поиск сущности в базе знаний по идентификатору. Нажмите на поле поиска.'
+                    },
+                    eng: {
+                        title: '',
+                        text: 'a'
+                    }
+                }
+            },
+            {
+                elemID: "#search-input",
+                desc: {
+                    rus: {
+                        title: 'Поиск по основному идентификатору',
+                        text: 'Это - поле поиска сущности по идентификатору. В него можно вводить как стандартные, так и системные идентификаторы. Давайте найдём сущность с основным идентификатором "транзитивное отношение".'
```

Comment:
maybe its not a good idea to save all this text here, maybe we can do it through knowledge base?